api/tests/test_security_user.py

```diff
@@ -0,0 +1,40 @@
+
+from .test_user import UserTestCase, client
+from ..models import SecurityUser
+
+
+class SecurityUserTestCase(UserTestCase):
+    def setUp(self):
+        UserTestCase.setUp(self)
+
+        self.security_users = "/api/v1/security_users/"
+
+        SecurityUser.objects.create(
+            email="sectest1@andela.com",
+            password="devpassword",
+            first_name="TestFirst",
+            last_name="TestLast",
+            phone_number="254720900900",
+            badge_number="AE23"
+        )
+        SecurityUser.objects.create(
+            email="sectest2@andela.com",
+            password="devpassword",
+            first_name="TestFirst2",
+            last_name="TestLast2",
+            phone_number="254720900900",
+            badge_number="AE24"
+        )
+
+    def test_admin_can_get_security_users_emails(self):
+        client.login(username='admin@site.com', password='devpassword')
```

Comment:
@HawiCaesar as mentioned on the ticket this endpoint is for use within the android codebase, not when a user is logged in. it will use an API key which we'll save as an env variable on both ends. lemme know if this makes sense.

Reply:
@egichuri it makes sense. But how will write a test for the endpoint? Or will the test work with the assumption of a fake key ?

Reply:
you should be able to test it once you set the authentication class(es) on the viewset.